integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb

```diff
@@ -6,8 +6,8 @@
 #Init
 master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
 moduledir = on(master, puppet('config', 'print', 'environmentpath')).stdout.strip + '/production/modules'
-git_environments_path = '/root/environments'
-last_commit = git_last_commit(master, git_environments_path)
+git_remote_environments_path = '/root/environments'
```

Comment:
Did "git" imply "remote" here before? I do like "remote" better but I'm wondering if we need "git" as part of the name?

Reply:
yeah I went back and forth on whether to leave "git" in or not. I opted for extra clarity around the fact that this is used to simulate committing to a control repo, but I can remove the "git" part if you don't like it.